storefront/src/app/modules/admin/custom/site/site-dashboard.component.scss

```
@@ -4,15 +4,38 @@
 }
 
 .spa-site-dashboard {
+    --spa-dashboard-cobalt: #2563eb;
+    --spa-dashboard-cobalt-strong: #1d4ed8;
+    --spa-dashboard-surface: rgba(255, 255, 255, 0.84);
+    --spa-dashboard-surface-strong: rgba(255, 255, 255, 0.94);
+    --spa-dashboard-border: rgba(191, 219, 254, 0.58);
+    --spa-dashboard-text: #0f172a;
+    color-scheme: light;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)